app/instrumentation-tests/src/org/commcare/androidTests/LogSubmissionTest.kt

```diff
@@ -42,4 +51,25 @@ class LogSubmissionTest: BaseTest() {
                 "Log submission was unsuccessful"
         )
     }
+
+    @Test
+    fun testReportProblem() {
+        InstrumentationUtility.selectOptionItem(withText("Advanced"))
+        onView(withText("Report Problem"))
+            .perform(click())
+        InstrumentationUtility.enterText(R.id.ReportText01,"This is a test for Report Problem")
+        InstrumentationUtility.hideKeyboard()
+        onView(withText("Submit Report")).isPresent()
+        InstrumentationUtility.rotateLeft()
+        InstrumentationUtility.matchTypedText(R.id.ReportText01,"This is a test for Report Problem")
+        InstrumentationUtility.hideKeyboard()
+        onView(withText("Submit Report")).isPresent()
+        InstrumentationUtility.rotatePortrait()
+        InstrumentationUtility.hideKeyboard()
+        onView(withText("Submit Report")).perform(click())
+        InstrumentationUtility.stubIntentWithAction(Intent.ACTION_SEND)
```

Comment:
we can remove this as it doesn't do anything here. This is only saying that if there is even an intent with action `Intent.ACTION_SEND`, return with the success result. It's not making sure that our intent actually had `Intent.ACTION_SEND`

Reply:
removed the step from the test.